content/en/profiler/connect_traces_and_profiles.md

```diff
@@ -75,13 +75,22 @@ os.Setenv("DD_PROFILING_EXECUTION_TRACE_PERIOD", "15m")
 
 Setting these variables will record up to 1 minute (or 5 MiB) of execution tracing data [every 15 minutes][2].
 
-While recording execution traces, your application may observe an increase in CPU usage similar to a garbage collection. Although this should not have a significant impact for most applications, Go 1.21 includes [patches][3] to eliminate this overhead.
+You can find this data:
+
+- In the [Profile List][3] by adding `go_execution_traced:yes` to your search query. Click on a profile to view the [Profile Timeline][4]. To go even deeper, download the profile and use `go tool trace` or [gotraceui][5] to view the contained `go.trace` files.
```

Comment:
Do we need to suggest using another tool here? Not that I'm opposed to it, those are both fine tools, it just seems to me that our documentation should focus on how to use our product. e.g. we don't suggest that Java users hop into JMC for in-depth JFR analysis, or to use `go tool pprof` to view pprof-formatted data locally, even though those are reasonable things to do sometimes.

Reply:
I agree that we should not try to explain how to use 3rd party tools for viewing our data. But we're not doing that, we're just pointing out their existence.

IMO this is the right thing to do here for a few reasons:

1. It really clarifies what kind of data we're collecting. There is a lot of public information you can find with these keywords.
2. It explains why we have a download feature. Our current documentation only mentions that we have it, but not how we expect people to use it.
3. It may help users to understand the data in more depth, and even make good suggestions for improvements we could make in our product.

On the other side, I don't see compelling reasons for withholding this information.

Reply:
The only reason to not include would be if it's out of scope for our documentation. I agree with what you said, just wanted to consider whether it's in scope. But, indeed it's good to know that our data is the same data you get straight from the runtime, that there's nothing proprietary or special about it, and that it's available if you need it :)

Reply:
Let's keep it as-is for now. Unless the docs team has a strong view on this, I think it's okay.